src/main/java/flab/delideli/service/CartServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package flab.delideli.service;
+
+import flab.delideli.dao.CartDao;
+import flab.delideli.dto.AddCartDTO;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class CartServiceImpl implements CartService{
```

Comment:
1. 요런곳에 컨벤션도 신경써주세요~ 
2. 아무리 잘하는 개발자라도 이렇게 깜빡하고 컨벤션이 잘못되는 경우는 흔할 것 같습니다. 이런 코드가 커밋되는건 막아버리고 싶은데 어떻게 하는게 좋을까요?

```suggestion
public class CartServiceImpl implements CartService {
```

Reply:
저도 모르는 부분이어서 찾아봤는데 `Git Hook`이라는 기능이 있는 것 같습니다. `Git Hook`은 특정 이벤트가 생겼을 때 특정 스크립트를 실행하는 기능이라고 합니다. `Git Hook`의 `pre-commit`에 컨벤션 검사 스크립트를 넣으면, 커밋할 때 컨벤션을 지킨 코드만 커밋되도록 설정할 수 있다고 합니다.